CHANGELOG.md

```diff
@@ -6,6 +6,11 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+### Fixed
+
+- **CUMULUS-544** - Post to CMR task has UAT URL hard-coded
+  - Changed to use getUrl. PostToCmr now requires CMR_ENVIRONMENT env to be specified if SIT or OPS are desired. Default is UAT.
```

Comment:
I would leave out the "Changed to use getUrl." part, since that's just an implementation detail that users of the task shouldn't be worried about.